internal/store/atlas/data_federation.go

```diff
@@ -22,7 +22,14 @@ import (
 	"go.mongodb.org/atlas-sdk/v20230201008/admin"
 )
 
-//go:generate mockgen -destination=../../mocks/atlas/mock_data_federation.go -package=atlas github.com/mongodb/mongodb-atlas-cli/internal/store/atlas DataFederationLister,DataFederationDescriber,DataFederationCreator,DataFederationUpdater,DataFederationDeleter,DataFederationLogDownloader
+//go:generate mockgen -destination=../../mocks/atlas/mock_data_federation.go -package=atlas github.com/mongodb/mongodb-atlas-cli/internal/store/atlas DataFederationLister,DataFederationDescriber,DataFederationStore,DataFederationCreator,DataFederationUpdater,DataFederationDeleter,DataFederationLogDownloader
+
+const NoTypeFlag = ""
```

Comment:
any reason for this e,pty string const?

Reply:
Will double check, thanks!

Reply:
I removed the const and also the extra arg in the interface, which we were never using anyway. Change comming up.